src/style.css

```diff
@@ -0,0 +1,113 @@
+* {
+  box-sizing: border-box;
+}
+
+p,
+h2,
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-size: 10px;
+  color: #616070;
+  --color-dark-blue: #060b35;
+}
+
+.page {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
```

Comment:
Maybe you don't need class-style for html and you can put styles by tag
And put this styles at the top of the file, it's easier to read